lib/main.dart

```diff
@@ -1,10 +1,13 @@
 import 'dart:convert';
-import 'dart:typed_data';
-import 'package:flutter/foundation.dart';
-import 'package:flutter/material.dart';
+
 import 'package:appwrite/appwrite.dart';
+import 'package:appwrite/models.dart';
 import 'package:file_picker/file_picker.dart';
-import 'package:appwrite/models.dart' as Models;
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:window_location_href/window_location_href.dart';
```

Comment:
what is this for?

Reply:
It allows us to get the current URL for the oauth callback on web. See

https://github.com/appwrite/playground-for-flutter/blob/9015a534de95438000a5c1c3b7065d7e7b7eb055/lib/main.dart#L61

and

https://github.com/appwrite/playground-for-flutter/blob/9015a534de95438000a5c1c3b7065d7e7b7eb055/lib/main.dart#L294